lib/representation/src/multitype.rs

```diff
@@ -36,49 +36,47 @@ pub fn extract_column_from_multitype(
     multitype_column: &Column,
     subtype: &BaseRDFNodeType,
 ) -> Column {
-    let mt_struct = multitype_column.struct_().unwrap();
     let colname = base_col_name(subtype);
     match subtype {
-        BaseRDFNodeType::Literal(_) if subtype.is_lang_string() => {
-            let mut lf = DataFrame::new(vec![
-                mt_struct
-                    .field_by_name(LANG_STRING_VALUE_FIELD)
-                    .unwrap()
-                    .cast(&DataType::String)
```

Comment:
NB: I'm no longer casting these to strings, if this is a bad idea I can add a thing to `filter_column` that lets you specify datatypes